.github/workflows/csharp-tests.yml

```diff
@@ -99,34 +106,25 @@ jobs:
     - uses: actions/download-artifact@v4
       with:
         pattern: ckzg-library-wrapper-*
-        path: bindings/csharp/Ckzg.Bindings/runtimes/linux-x64/native
-    - uses: actions/download-artifact@v4
-      with:
-        pattern: ckzg-library-wrapper-*
-        path: bindings/csharp/Ckzg.Bindings/runtimes/osx-x64/native
-    - uses: actions/download-artifact@v4
-      with:
-        pattern: ckzg-library-wrapper-*
-        path: bindings/csharp/Ckzg.Bindings/runtimes/win-x64/native
-    - uses: actions/download-artifact@v4
-      with:
-        pattern: ckzg-library-wrapper-*
-        path: bindings/csharp/Ckzg.Bindings/runtimes/osx-arm64/native
-    - uses: actions/download-artifact@v4
-      with:
-        pattern: ckzg-library-wrapper-*
-        path: bindings/csharp/Ckzg.Bindings/runtimes/linux-arm64/native
```

Comment:
These actions were downloading all 5 artifacts 5 times for no reason. Now, it does once and later puts each of them where they should be.